### ISSUE TITLE
Default pattern for Duration JsonConverter is different from the standard one

### ISSUE THREAD

**Issue by maciejjarzynski** [nodatime/nodatime.serialization#27]
The standard Duration pattern described in the docs:
https://nodatime.org/2.3.x/userguide/duration-patterns
is `-D:hh:mm:ss.FFFFFFFFF`
whereas the `JsonConverter` in 
https://github.com/nodatime/nodatime.serialization/blob/edf88349e54ab63f06266a9c864edfa1ba6503cb/src/NodaTime.Serialization.JsonNet/NodaConverters.cs#L102

has a pattern `-H:mm:ss.FFFFFFFFF`.
Does this serve any purpose, or it can simply be `DurationPattern.Roundtrip` there?

**Comment by jskeet** [comment]
I honestly don't remember the origin of using this instead of the standard pattern, but I'd argue that we shouldn't change it at this point; it would be a breaking change for stored information for little benefit.

Rather than change either, I think it would probably be more useful to add an extra standard pattern that corresponds with the JSON one. Does that sound like a reasonable compromise?

**Comment by maciejjarzynski** [comment]
Yes, for the sake of compatibility that would be an option, thanks ;)

**Comment by jskeet** [comment]
Righto - renamed the issue for what it's turned into just for my convenience :)
